mistletoe/html_renderer.py

```diff
@@ -18,13 +18,19 @@ class HTMLRenderer(BaseRenderer):
 
     See mistletoe.base_renderer module for more info.
     """
-    def __init__(self, *extras):
+    def __init__(self, *extras, html_escape_double_quotes=True, html_escape_single_quotes=False):
```

Comment:
Should the `HTMLRenderer.init()` perhaps also take **kwargs parameters? And `Scheme`, too?

Reply:
Done.

Reply:
Oddly enough, I can see now that I have somehow overlooked to also extend the constructor of `PygmentsRenderer` - will do additionally...